pom.xml

```diff
@@ -143,7 +143,7 @@
         <spark.hofs.version>0.4.0</spark.hofs.version>
         <spark.hats.version>0.2.1</spark.hats.version>
         <typesafe.config.version>1.3.1</typesafe.config.version>
-        <spline.version>0.3.9</spline.version>
+        <spline.version>0.4.1</spline.version>
```

Comment:
Reminder before the final merge. Spline released 0.4.2. Might release more fixes by the time of merge.